workflow/scripts/update_population_db.py

```diff
@@ -0,0 +1,78 @@
+import os
+import shutil
+import sys
+import tempfile
+
+sys.stderr = open(snakemake.log[0], "w")
+
+import subprocess as sp
+import pysam
+
+db_path = snakemake.output[0]
+
+
+def sample_in_db(group):
+    samples = pysam.VariantFile(db_path).header.samples
+    if group in samples:
+        if len(samples) == 1:
+            return "only"
+        else:
+            return "one"
+    else:
+        return "not"
+
+
+def copy_to_db(bcf_path):
+    shutil.copyfile(bcf_path, db_path)
+
+
+def update_db(bcf_path, group):
+    sp.run(["bcftools", "index", "-f", db_path])
+
+    match sample_in_db(group):
+        case "only":
+            copy_to_db(bcf_path)
+            return
+        case "one":
```

Comment:
Can this be solved differently?
In case the sample already exists it needs to be removed from the population db before merging.
`bcftools merge` does not offer a parameter for omitting samples